packages/block-editor/src/components/editable-text/README.md

```diff
@@ -1,4 +1,4 @@
-# `Editable`
```

Comment:
Not sure if I should move these docs to plain text, even though it's an experimental v2.